Makefile

```diff
@@ -2,6 +2,7 @@ version := $(shell python3 -c 'from suse_migration_services.version import __VER
 
 build: check test
 	rm -f dist/*
+        # copy in the correct MANIFEST.in
```

Comment:
This line uses expanded tab, but the other lines use tab, so the indentation is not consistent

Reply:
Thanks, that was not supposed to be there, it was leftover from exploring different options to resolve the "conflicts issue'